Simulation/Main.m

```diff
@@ -1,30 +1,36 @@
-% Base off of https://github.com/TSoGiants/MECH2103-Computational-Aerodynamics
+% Based off of https://github.com/TSoGiants/MECH2103-Computational-Aerodynamics
 
 % Object parameters
-Plane.Mass = 1;
+Plane.Mass        = 1; % Mass (kg)
+Plane.DragRefArea = 1; % Cross sectional area used for drag (m2)
+Plane.LiftRefArea = 0; % Projected wing area used for lift (m2)
```

Comment:
Are they really the same value? I know that reference area for drag is the cross sectional area of the plane but isn't the reference area for lift the actual area of the wings, because that's what matters for lift?

Reply:
They are the same value. This is needed so that they can be compared equally. The choice for what the reference area will be arbitrary since it only affects the scale of the Cd and Cl but by convention things like "Top down" area or "Front facing" area is used so that the scales across multiple types of aircraft/car/rocket designs are relatively equal.

Reply:
The change has been made in 06a0894507f717b10d2412fbf40733a744029dae.